e3nn_jax/_src/s2grid.py

```diff
@@ -158,7 +158,7 @@ def spherical_harmonics_s2_grid(lmax: int, res_beta: int, res_alpha: int, *, qua
 
 
 def from_s2grid(
-    x: jnp.ndarray, lmax: int, normalization="component", lmax_in=None, *, quadrature: str, p_val: int, p_arg: int
+    x: jnp.ndarray, lmax: int, normalization="component", lmax_in=None, *, quadrature: str, naive=False, p_val: int, p_arg: int
```

Comment:
Maybe we can call the argument `fft=True` instead of `naive=False`. I think it would be more explicit